[zxing-cpp] `find_package(zxing)` does not work

From https://github.com/microsoft/vcpkg/pull/4854#issuecomment-495631526:

> @ras0219-msft Could it be that the renaming zxing -> zxing-cpp led to some errors?
> 
> I am using the CMake integration
> 
> set(CMAKE_TOOLCHAIN_FILE "D:/DEVELOPMENT/vcpkg/scripts/buildsystems/vcpkg.cmake" CACHE STRING "")
> and currently get the CMake error:
> 
> CMake Error at D:/DEVELOPMENT/vcpkg/installed/x64-windows/share/zxing/zxing-targets.cmake:80 (message):
>   The imported target "zxing::zxing" references the file
> 
>      "D:/DEVELOPMENT/vcpkg/installed/x64-windows/tools/zxing-cpp/zxing.exe"
> 
>   but this file does not exist.
> The slightly different file vcpkg/installed/x64-windows/tools/zxing-cpp/zxing.exe (with the -cpp) does exist.
> 
> Additionally, vcpkg\installed\x64-windows\share\zxing-cpp contains the copyright notice while vcpkg\installed\x64-windows\share\zxing contains the cmake scripts (zxing-config.cmake, zxing-targets.cmake, ...)
> 
> And the last question: What is the .exe for? Does my application need it as well?

--- a/ports/zxing-cpp/CONTROL
+++ b/ports/zxing-cpp/CONTROL
@@ -1,4 +1,4 @@
 Source: zxing-cpp
-Version: 3.3.3-3
+Version: 3.3.3-4
 Build-Depends: opencv[core]
 Description: Barcode detection and decoding library.

--- a/ports/zxing-cpp/portfile.cmake
+++ b/ports/zxing-cpp/portfile.cmake
@@ -22,13 +22,13 @@ vcpkg_install_cmake()
 
 vcpkg_fixup_cmake_targets(CONFIG_PATH "lib/zxing/cmake" TARGET_PATH share/zxing)
 
-file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/tools/zxing)
+file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/tools/zxing-cpp)
 if (NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
-    file(COPY ${CURRENT_PACKAGES_DIR}/bin/zxing.exe DESTINATION ${CURRENT_PACKAGES_DIR}/tools/zxing)
+    file(COPY ${CURRENT_PACKAGES_DIR}/bin/zxing.exe DESTINATION ${CURRENT_PACKAGES_DIR}/tools/zxing-cpp)
 else()
-    file(COPY ${CURRENT_PACKAGES_DIR}/bin/zxing DESTINATION ${CURRENT_PACKAGES_DIR}/tools/zxing)
+    file(COPY ${CURRENT_PACKAGES_DIR}/bin/zxing DESTINATION ${CURRENT_PACKAGES_DIR}/tools/zxing-cpp)
 endif()
-vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/zxing)
+vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/zxing-cpp)
 
 vcpkg_copy_pdbs()
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin)

